sqlmesh/core/test/definition.py

```diff
@@ -241,12 +250,13 @@ def test_ctes(self, ctes: t.Dict[str, exp.Expression]) -> None:
                     )
 
                 cte_query = ctes[cte_name].this
+                query_has_orderby = any(cte_query.find_all(exp.Order))
```

Comment:
the order by should only be on the final projection query right?

this checks for order by on ANY part of the sql qurey

Reply:
I believe you can either check `args.get("order")` or try `find_all_in_scope(exp.Order)`